apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -325,15 +333,17 @@ export default class SetupChecklist extends Component {
             {!linuxPermissionError && this.installFirmwareSketch()}
             {this.contactSupport()}
           </ValidationStep>
-          <ValidationStep
-            stepStatus={this.state[STATUS_BOARD_COMPONENTS]}
-            stepName={i18n.validationStepBoardComponentsUsable()}
-          >
-            {applabI18n.makerSetupVerifyComponents()}
-            <br />
-            {this.installFirmwareSketch()}
-            {this.contactSupport()}
-          </ValidationStep>
+          {this.state.boardTypeDetected !== BOARD_TYPE.MICROBIT && (
```

Comment:
it feels a little confusing to have this step render then disappear once we know it's a micro:bit. could we invert this logic and only show this step for boards that should have it? alternatively, we might be able to check that `boardTypeDetected && boardTypeDetected !== BOARD_TYPE.MICROBIT` to get the same outcome

Reply:
@KylieModen - I'd love your input here. I think Maddie's point makes sense, but that would change the current behavior for CP, vs. this change which keeps the current UI the same for CP, but changes for MB. Thoughts?

Reply:
I agree the flashing feels weird - I also don't love the order changing for what gets the thumbs up 

Is there any other way to not display anything initially to avoid the flash? that way CP users see what they always have, and MB users will only see their UX?

Reply:
That's a tough one - the board is detected at the second step - so we don't know whether we need 3 or 4 steps until them. So we could display only each step when it's completed and not the "pending" grey step?

Reply:
Mocked it up so we could see what that might look like:
![4stepLoad](https://user-images.githubusercontent.com/2959170/155821352-e53e1698-f67e-496d-9abf-2e79cfb17441.gif)
![3stepload](https://user-images.githubusercontent.com/2959170/155821360-d423369b-5303-4277-8431-45d81839c169.gif)

Reply:
love this update, LGTM